<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QMainWindow" name="MainMenu">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>394</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>612</width>
    <height>394</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>612</width>
    <height>394</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Главное меню</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>611</width>
      <height>371</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="userDataLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="createNewGameButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Создать новую игру</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createClanButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Создать клан</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showClansButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>180</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Показать кланы</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createTournamentButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Создать турнир</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>251</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="friendsTab">
      <attribute name="title">
       <string>Друзья</string>
      </attribute>
      <widget class="QListWidget" name="friendsListWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>241</width>
         <height>241</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="friendsRequestsTab">
      <attribute name="title">
       <string>Заявки в друзья</string>
      </attribute>
      <widget class="QListWidget" name="friendRequestsListWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>241</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="findGameButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Найти игру</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openShopButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>230</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Открыть магазин</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startGameWithAIButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Игра с ботом</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openTopPlayersWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>ТОП игроков</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>611</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:black</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QToolButton" name="exitButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>0</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
background: red;</string>
      </property>
      <property name="text">
       <string>×</string>
      </property>
     </widget>
     <widget class="QToolButton" name="hideButton">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>0</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
background: lightblue;</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="addFriendButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>320</y>
       <width>251</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить друга</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>612</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
